templates/flavors/k3s/full-vpcless/kustomization.yaml

```diff
@@ -178,3 +179,20 @@ patches:
     patch: |-
       - op: remove
         path: /spec/template/spec/firewallRef
+  - target:
+      group: infrastructure.cluster.x-k8s.io
+      version: v1alpha2
+      kind: LinodeMachineTemplate
+      name: .*-control-plane
+    patch: |-
+      apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+      kind: LinodeMachineTemplate
+      metadata:
+        name: ${CLUSTER_NAME}-control-plane
+      spec:
+        template:
+          spec:
+            placementGroupRef:
+              apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+              kind: LinodePlacementGroup
+              name: ${CLUSTER_NAME}
```

Comment:
question: do we need to add this for worker machines too? or is that not the intention for placement groups?

Reply:
Nope, it's only for the control-plane nodes. I've updated that in the latest commit